samples/RazorSlices.Samples.WebApp/Program.cs

```diff
@@ -85,6 +86,37 @@
         : TypedResults.NotFound();
 });
 
+
+var htmxTodos = app.MapGroup("/htmx-todo");
+var renderTodoTable = () => Results.Extensions.RazorSlice<Slices.HtmxTodos._Todos, List<Models.Todo>>(Models.Todos.AllTodos);
+htmxTodos.MapGet("/", () => Results.Extensions.RazorSlice<Slices.HtmxTodos.TodoIndex, List<Models.Todo>>(Models.Todos.AllTodos));
+htmxTodos.MapPost("/", ([FromForm] string title) => {
```

Comment:
Is it normal to return directly likes this from a POST when using HTMX, rather than doing a Post-Redirect-Get?